pkg/apis/kops/componentconfig.go

```diff
@@ -184,10 +184,10 @@ type KubeAPIServerConfig struct {
 	CloudProvider string `json:"cloudProvider,omitempty" flag:"cloud-provider"`
 	// SecurePort is the port the kube runs on
 	SecurePort int32 `json:"securePort,omitempty" flag:"secure-port"`
-	// InsecurePort is the port the insecure api runs
-	InsecurePort int32 `json:"insecurePort,omitempty" flag:"insecure-port"`
-	// Address is the binding address for the kube api
-	Address string `json:"address,omitempty" flag:"address"`
+	// Deprecated: InsecurePort is the port the insecure api runs
+	InsecurePort int32 `json:"insecurePort,omitempty" flag:"insecure-port" flag-empty:"-1"`
+	// Deprecated: Address is the binding address for the kube api
+	Address string `json:"address,omitempty" flag:"address" flag-empty:"255.255.255.255"`
```

Comment:
Similarly, can we keep empty-string as the flag-empty value?  Just seems more intuitive to have the empty values be the default uninitialized values in go as well.  Where that doesn't work is when 0 or "" is a valid flag value...

Reply:
ok, "" is ok. I just am not familiar with this code, I thought it may has validation.